cmd/buildah/images.go

```diff
@@ -5,8 +5,14 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/urfave/cli"
+	"encoding/json"
 )
 
+type JsonImage struct {
+	Id string `json:"id"`
```

Comment:
For the consumer, I think best to allow the key to remain but have the value be ""  ... then the consumer wont get key errors with simple iteration.  Will fix the Id casing.